ci_env.yml

```
@@ -0,0 +1,15 @@
+name: biomechanical-analysis-framework
+channel:
+  - conda-forge
+dependencies:
+  - cmake
+  - compilers
+  - make
+  - ninja
+  - pkg-config
+  - yarp
+  - idyntree
+  - spdlog
+  - catch2>=3.3.0
+  - manif
+  - bipedal-locomotion-framework
```

```suggestion
  - bipedal-locomotion-framework
```

Either you install it from source, or you install it from conda. I would not install it from conda and then override it with a different version compiled from source, this could create problems.

---

I install `blf` from source on ubuntu while I install it from conda on windows and mac; I saw that also for `human-dynamics-estimation`, from which the action is derived, is done like this.

---

But in human-dynamics-estimation the Linux CI uses apt to install dependencies, and so you can't install blf via apt. If here you are installing dependencies using conda in all operating systems, why you are not installing blf as well from conda?

---

I can install the dependencies from mamba also for ubuntu then